src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Migration/v0.04/01-setup-tables.sql

```diff
@@ -0,0 +1,50 @@
+-- Enum: delegation.delegationChangeType
+DO $$ BEGIN
+    CREATE TYPE delegation.delegationChangeType AS ENUM ('grant', 'revoke', 'revokelast');
+EXCEPTION
+    WHEN duplicate_object THEN null;
+END $$;
+
+-- Table: drop/create delegation.delegationChanges table both to drop old content and change from isDeleted to delegationChangeType
+DROP TABLE IF EXISTS delegation.delegationChanges CASCADE;
```

Comment:
@elsand 
I just realised I basically have no clue how these scripts are executed for a controlled incremental DB deploy, and if this is at all safe.

Are they executed only once, or for each and every deploy?
They aren't executed every time in Dev at least.

Where and/or how is it keeping track of which deploy scripts have been executed?

Reply:
@jonkjetiloye The state is kept in the `public`-schema in a table `__yuniql_schema_version`, so as long as that data isn't purged we're good.

We probably should consolidate all the migrations at some point though, if only to keep everything readable.